packages/react-instantsearch-dom-maps/src/GeoSearchContext.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type GeoSearchContextProps = {
+  isRefineOnMapMove: boolean;
+  hasMapMoveSinceLastRefine: boolean;
+  toggleRefineOnMapMove: () => void;
+  setMapMoveSinceLastRefine: () => void;
+  refineWithInstance: () => void;
```

Comment:
The function takes the map instance as argument: `(value: google.maps.Map) => void`